src/main/java/wile/engineersdecor/blocks/EdHatchBlock.java

```diff
@@ -78,7 +75,7 @@ public RenderTypeHint getRenderTypeHint()
 
   @Override
   public VoxelShape getShape(BlockState state, BlockGetter source, BlockPos pos, CollisionContext selectionContext)
-  { return state.getValue(OPEN) ? vshapes_open.get((state.getValue(HORIZONTAL_FACING)).get3DDataValue() & 0x7) : super.getShape(state, source, pos, selectionContext); }
+  { return state.getValue(OPEN) ? vshapes_open.get((state.getValue(HORIZONTAL_FACING)).get3DDataValue()) : super.getShape(state, source, pos, selectionContext); }
```

Comment:
Not sure why this is needed, maybe I am missing something obvious

Reply:
Oh, that is not strictly needed - it's just a guarantee that no invalid direct array index accesses are made in case something changes in the value range.